packages/AutoTDD-GUI.package/ATDDGitHubActionResults.class/instance/shouldSimplifySmalltalkCiLog..st

```diff
@@ -0,0 +1,4 @@
+accessing
+shouldSimplifySmalltalkCiLog: aBoolean
+	
+	shouldSimplifySmalltalkCiLog := aBoolean
```

Comment:
```suggestion
accessing
shouldSimplifyLog: aBoolean
	
	shouldSimplifyLog := aBoolean
```
I felt the names were a little too long to support readability. Although the parsing is in the same class i would not include any info on implementation (SmalltalkCiLog vs Log) in GUI code